### ISSUE TITLE
invalid memory address or nil pointer dereference (UpdateUserNick)

### ISSUE THREAD

**Issue by faljse** [42wim/matterircd#141]
Got this multiple times today.
There are a lot of failed login attempts in my log - so this could be the result of malicious input.

```
17:41:56 panic: runtime error: invalid memory address or nil pointer dereference
17:41:56 [signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x805b26]
17:41:56 goroutine 2547 [running]:
17:41:56 github.com/42wim/matterircd/vendor/github.com/42wim/matterbridge/matterclient.(*MMClient).UpdateUserNick(0x0, 0xc4204866c5, 0x3, 0x97ac65, 0x40d57e)
17:41:56         /go/src/github.com/42wim/matterircd/vendor/github.com/42wim/matterbridge/matterclient/matterclient.go:595 +0x26
17:41:56 github.com/42wim/matterircd/vendor/github.com/42wim/mm-go-irckit.CmdNick(0xc0aae0, 0xc4204888f0, 0xc420394620, 0xc42019caa0, 0xc4201cd128, 0xc420434e01)
17:41:56         /go/src/github.com/42wim/matterircd/vendor/github.com/42wim/mm-go-irckit/server_commands.go:268 +0x63
17:41:56 github.com/42wim/matterircd/vendor/github.com/42wim/mm-go-irckit.commands.Run(0xc4204f4f30, 0xc0aae0, 0xc4204888f0, 0xc420394620, 0xc42019caa0, 0x0, 0x0)
17:41:56 matterircd.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
17:41:56 matterircd.service: Unit entered failed state.
```

sh-4.4# /opt/matterircd/matterircd -version
version: 0.16.2 21dd4d1
(pre-built binary from github/releases)

sh-4.4# uname -a
Linux mattermost 4.14.11-200.fc26.x86_64 #1 SMP Wed Jan 3 13:58:53 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

**Comment by 42wim** [comment]
Thanks for reporting
I've released a 0.16.3 which fixes this issue

**Comment by faljse** [comment]
Wow, that was fast! :)
Thx!
